pom.xml

```diff
@@ -170,6 +170,11 @@
             <scope>test</scope>
             <version>3.0.3</version>
         </dependency>
+        <dependency>
```

Comment:
The library doesn't need MSAL dependency and it is required by user's application when implementing the OAuth callback, right?